frontend/src/ui/expenseCategories.js

```diff
@@ -39,6 +39,24 @@ export default function(groupid) {
         'Serveur',
         'Transport'
       ];
+       break;
+    case: 114: // partidodigital
```

Comment:
@piamancini Remove the colon `:` after the word `case`

So:
`case: 114: // partidogital` 
becomes 
`case 114: // partidogital`